airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/base/spec_modification/SpecModifyingDestination.java

```diff
@@ -41,4 +42,12 @@ public AirbyteMessageConsumer getConsumer(final JsonNode config,
     return destination.getConsumer(config, catalog, outputRecordCollector);
   }
 
+  @Override
+  public SerializedAirbyteMessageConsumer getSerializedMessageConsumer(final JsonNode config,
```

Comment:
this isn't used by redshift, just adding it since I'm in this part of the code anyway